kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/projectExt/configureThirdPartyPlugins.kt

```diff
@@ -1,20 +1,17 @@
 package godot.gradle.projectExt
 
 import com.github.jengelman.gradle.plugins.shadow.ShadowPlugin
-import com.google.devtools.ksp.gradle.KspGradleSubplugin
-import godot.tools.common.constants.FileExtensions
 import org.gradle.api.Project
+import org.gradle.api.plugins.scala.ScalaPlugin
 import org.gradle.plugins.ide.idea.IdeaPlugin
 import org.jetbrains.gradle.ext.IdeaExtPlugin
-import java.io.File
 
 fun Project.configureThirdPartyPlugins() {
     //apply needed third party plugins
-    repositories.google() //add google repository for ksp (kotlin symbol processing
-    pluginManager.apply(KspGradleSubplugin::class.java)
-    pluginManager.apply(IdeaPlugin::class.java) //needed so idea can find and index the generated sources from ksp
-    pluginManager.apply(IdeaExtPlugin::class.java) //needed so idea can find and index the generated sources from ksp
+    pluginManager.apply(IdeaPlugin::class.java) //needed so idea can find and index the generated sources from classgraph
+    pluginManager.apply(IdeaExtPlugin::class.java) //needed so idea can find and index the generated sources from classgraph
     pluginManager.apply(ShadowPlugin::class.java)
+    pluginManager.apply(ScalaPlugin::class.java)
```

Comment:
I'd rather not add scala support out of the box. I'd rather provide a configuration where scala support can be enabled.

Reply:
I don't really know how to do so, since configurations are not resolved before plugins

Reply:
I already prepared it in a separate branch to be merged after this one. Will resolve the conversation :-)